bootstrap/src/uri/dsm-uri.ts

```diff
@@ -113,23 +113,27 @@ export class DsmUri implements ZLUX.UriBroker {
    */
   pluginConfigForScopeUri(pluginDefinition: ZLUX.Plugin, scope: string, resourcePath:string, resourceName?:string): string {
```

Comment:
Add version as an actual optional parameter, but default to _current. Give people that option to specify. In the future, the uriBroker can be clever in taking different parameters or handling them differently according to versions.

Reply:
I'd think including the config service, actually, but we can probably get away with _current for it for now too.
Yes, everything that could have _current can also have a specific version, and therefore it should be a parameter, even if its unlikely people would use it.